packages/chaire-lib-common/src/services/geodata/ManageOverlappingLines.ts

```diff
@@ -1,58 +1,48 @@
-import { lineOffset, lineOverlap, lineString, LineString } from '@turf/turf';
+/*
+ * Copyright 2022, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { lineOffset, lineOverlap, lineString } from '@turf/turf';
+import { LineString } from 'geojson';
+
+export const OFFSET_WIDTH = 3; // Offset of each line in meters
 
 interface OverlappingSegments {
+    /**
+     * GeoJSON data containing the overlapping coordinates
+     */
     geoData: GeoJSON.Feature<LineString>;
+    /**
+     * IDs of each line in conflict with the specified overlap
```

Comment:
IDs of each feature? As in the `id` field of the feature object?

Reply:
yes, feature.id